### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by warfollowsme** [Grepsy/BinanceService#1]
I re-write some of your code and create [API Binance wrapper](https://github.com/WarFollowsMe/NetBinance) 
Maybe it will be useful to you.

**Comment by Grepsy** [comment]
Hi, I'm using this code for the Binance API competition. So it's fine if you want to re-use it for a personal project. It's not fine if you use/submit it for the competition.

**Comment by warfollowsme** [comment]
If everyone starts to create their own wrapper for competition, imagine how many shit code with label 'binance' will appear. I just suggest using some good base wrapper for all projects.

**Comment by Grepsy** [comment]
The way I see it an elegant wrapper is a large part of the competition and to show example usage of the API. I don't mind sharing that code and taking improvements once the competition is done, but until then I'm not into sharing that, as it won't work for the competition.

Besides that I don't appreciate you announcing `hello. i'm write .net wrapper for api https://github.com/WarFollowsMe/NetBinance I'm just started but general, marketdata and account endpoints already done.` while a large part of that was my work.

**Comment by warfollowsme** [comment]
Surely not large part. Just compare my and your code. But anyway, I just trying to help. Please don't have hard feelings.
